### PR TITLE
Update modules and plugins to the latest versions.

### PR DESCRIPTION
**Pull request by javadev**

### CODE REVIEW

**Comment by dmlittle** [comment]
LGTM

**Comment by robinjoseph08** [comment]
LGTM 👍

**Comment by javadev** [comment]
Please create your own verioneye check for the pom.xml for the lob/lob-java repository and mute some libs which is not compatible with java 6.
screenshot_2016-07-16-02-24-20_com android chrome

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 [![Maven Central](https://img.shields.io/maven-central/v/com.lob/lob-java.svg)](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.lob%22%20AND%20a%3A%22lob-java%22)
 [![Build Status](https://secure.travis-ci.org/lob/lob-java.svg)](https://travis-ci.org/lob/lob-java)
 [![Coverage Status](https://coveralls.io/repos/lob/lob-java/badge.svg?branch=master)](https://coveralls.io/r/lob/lob-java)
+[![Dependency Status](https://www.versioneye.com/user/projects/5788653cc3d40f0046851db1/badge.svg?style=flat)](https://www.versioneye.com/user/projects/5788653cc3d40f0046851db1)
```
Comment (javadev) [inline README.md]: It points to the javadev/lob-java repository.

**Pull request merged**